operating_unit/README.rst

```diff
@@ -1,72 +1,106 @@
-   :target: https://runbot.odoo-community.org/runbot/213/11.0
-
-Bug Tracker
-===========
-
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/operating-unit/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smash it by providing detailed and welcomed feedback.
-
-Credits
-=======
-
-Images
-------
-
-* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
-
-Contributors
-------------
-
-* Eficent <contact@eficent.com>
-* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
-
-Maintainer
-----------
-
-.. image:: https://odoo-community.org/logo.png
-   :alt: Odoo Community Association
-   :target: http://odoo-community.org
-
-This module is maintained by the OCA.
-
-OCA, or the Odoo Community Association, is a nonprofit organization whose
-mission is to support the collaborative development of Odoo features and
-promote its widespread use.
-
-To contribute to this module, please visit http://odoo-community.org.
+
+==============
+Operating Unit
+==============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--tools-lightgray.png?logo=github
+    :target: https://github.com/OCA/operating_unit/tree/12.0/operating_unit
+    :alt: OCA/operating_unit
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-tools-11-0/server-tools-11-0-kpi
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/253/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3|
+
+An operating unit (OU) is an organizational entity part of a company, with
+separate management ownership. Management by OU is aimed to introduce the
+following features:
+
+* Data partitioning from other OU.
+* Own sequencing schemes.
+* User access administration for data processing and reporting.
+* Product and customer independence of OU.
+* OU specific P&L and Balance sheet.
+
+
+**Table of contents**
+
+.. contents::
+   :local:	
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Assign *Multi Operating Unit* group to user.
+* Go to *Settings / Users & Companies / Operating Units* and create Operating Units.

```

Comment:
this needs to be added to a new fragment CONFIGURE.rst. See https://github.com/OCA/maintainer-tools/tree/master/template/module/readme